backends/s3/src/main/scala/geotrellis/s3/S3Client.scala

```diff
@@ -14,10 +14,10 @@ import com.amazonaws.ClientConfiguration
 import org.apache.commons.io.IOUtils
 import scala.collection.JavaConversions._
 
-trait S3Client extends LazyLogging {
+trait S3Client extends LazyLogging with Serializable {
```

Comment:
Ah, okay.  That was to enable to `saveToS3` method to work in the context of a unit test.  I wanted to be able to switch between a mock S3 and a real one.